pkg/vpcmodel/abstractVPC.go

```diff
@@ -8,6 +8,8 @@ import (
 type VPCResourceIntf interface {
 	UID() string
 	Name() string
+	Names() (string, []string)
+	Type() string
```

Comment:
It's unclear for the reader what's the difference between `Name()` and `Names()` and why we need both. Please add a comment

Reply:
done